packages/arg-parser/src/arg-parser.ts

```diff
@@ -1,125 +1,313 @@
+    }),
+    sslDisabledProtocols: z.string().optional().register(cliOptionsRegistry, {
+      deprecationReplacement: 'tlsDisabledProtocols',
+    }),
+
+    // TLS options
+    tlsCAFile: z.string().optional(),
+    tlsCertificateKeyFile: z.string().optional(),
+    tlsCertificateKeyFilePassword: z.string().optional(),
+    tlsCertificateSelector: z.string().optional(),
+    tlsCRLFile: z.string().optional(),
+    tlsDisabledProtocols: z.string().optional(),
+
+    // Boolean options
+    apiDeprecationErrors: z.boolean().optional(),
+    apiStrict: z.boolean().optional(),
+    buildInfo: z
+      .boolean()
+      .optional()
+      .register(cliOptionsRegistry, { alias: ['build-info'] }),
+    exposeAsyncRewriter: z.boolean().optional(),
+    help: z
+      .boolean()
+      .optional()
+      .register(cliOptionsRegistry, { alias: ['h'] }),
+    ipv6: z.boolean().optional(),
+    nodb: z.boolean().optional(),
+    norc: z.boolean().optional(),
+    oidcTrustedEndpoint: z.boolean().optional(),
+    oidcIdTokenAsAccessToken: z
+      .boolean()
+      .optional()
+      .register(cliOptionsRegistry, {
+        alias: ['oidcIDTokenAsAccessToken'],
+      }),
+    oidcNoNonce: z.boolean().optional(),
+    perfTests: z.boolean().optional(),
+    quiet: z.boolean().optional(),
+    retryWrites: z.boolean().optional(),
+    shell: z.boolean().optional(),
+    smokeTests: z.boolean().optional(),
+    skipStartupWarnings: z.boolean().optional(),
+    verbose: z.boolean().optional(),
+    version: z.boolean().optional(),
+
+    // Deprecated SSL boolean options
+    ssl: z
+      .boolean()
+      .optional()
+      .register(cliOptionsRegistry, { deprecationReplacement: 'tls' }),
+    sslAllowInvalidCertificates: z
+      .boolean()
+      .optional()
+      .register(cliOptionsRegistry, {
+        deprecationReplacement: 'tlsAllowInvalidCertificates',
+      }),
+    sslAllowInvalidHostnames: z
+      .boolean()
+      .optional()
+      .register(cliOptionsRegistry, {
+        deprecationReplacement: 'tlsAllowInvalidHostnames',
+      }),
+    sslFIPSMode: z.boolean().optional().register(cliOptionsRegistry, {
+      deprecationReplacement: 'tlsFIPSMode',
+      unsupported: true,
+    }),
+
+    // TLS boolean options
+    tls: z.boolean().optional(),
+    tlsAllowInvalidCertificates: z.boolean().optional(),
+    tlsAllowInvalidHostnames: z.boolean().optional(),
+    tlsFIPSMode: z.boolean().optional(),
+    tlsUseSystemCA: z.boolean().optional(),
+
+    // Array options
+    eval: z.array(z.string()).optional(),
+    file: z
+      .array(z.string())
+      .optional()
+      .register(cliOptionsRegistry, { alias: ['f'] }),
+
+    // Options that can be boolean or string
+    json: z.union([z.boolean(), z.enum(['relaxed', 'canonical'])]).optional(),
+    oidcDumpTokens: z
+      .union([z.boolean(), z.enum(['redacted', 'include-secrets'])])
+      .optional(),
+    browser: z.union([z.boolean(), z.string()]).optional(),
```

Comment:
Fwiw we wouldn't want to accept `true` as a boolean, i.e. `--browser=true` should be interpreted as literally referring to the `true` UNIX command line utility – `false` is the only special value that's supposed to be interpreted as not attempting to spawn an executable at all

Reply:
ah okay, so false is valid but not true? Is this unique to `brower` or all others?